reoptjl/test/test_job_endpoint.py

```diff
@@ -60,11 +60,11 @@ def test_multiple_outages(self):
         self.assertAlmostEqual(results["Outages"]["expected_outage_cost"], 0.0, places=-2)
         self.assertAlmostEqual(sum(sum(np.array(results["Outages"]["unserved_load_per_outage_kwh"]))), 0.0, places=0)
         self.assertAlmostEqual(results["Outages"]["microgrid_upgrade_capital_cost"], 1927766, places=-2)
-        self.assertAlmostEqual(results["Financial"]["lcc"], 59597421, places=-3)
+        self.assertAlmostEqual(results["Financial"]["lcc"], 59658835, places=-3)
 
     def test_pv_battery_and_emissions_defaults_from_julia(self):
         """
-        Same test post as"Solar and Storage w/BAU" in the Julia package. Used in development of v3.
+        Same test post as"Solar and ElectricStorage w/BAU" in the Julia package. Used in development of v3.
```

Comment:
If editing the docstrings anyhow, I'll recommend changing  `as"Solar and ElectricStorage w/BAU"` to `as "Solar and ElectricStorage w/BAU"` - i.e., add a space after "as"

Reply:
I implemented this in https://github.com/NREL/REopt_API/pull/487/commits/2dbdc27bd221c69bd92f0d0af4a2b28315e3f26a, feel free to merge if you agree